app/editor/src/features/admin/sources/DataSourceLists.tsx

```diff
@@ -35,6 +37,7 @@ export const DataSourceList: React.FC<IDataSourceListProps> = (props) => {
         columns={dataSourceColumns}
         isLoading={!!requests.length}
         sorting={{ sortBy: [{ id: 'id', desc: false }] }}
+        manualPageSize={keycloak.hasClaim(Claim.administrator)}
```

Comment:
Same comment as above, may not be necessary?

Reply:
Not needed